From 4b6f6384246c976f3d6ab3ea29b0a6afe83e840e Mon Sep 17 00:00:00 2001
From: Juno Computers <thelookouts@hotmail.com>
Date: Sat, 26 Aug 2023 09:35:57 -0400
Subject: [PATCH] Update 60-sensor.hwdb (#28804)

Updated T11/Juno Tab 2.

(cherry picked from commit 8a76d3cc7029ea9add4d4ed38af72745f6ba7974)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 4fb7387f60..6bcf2cba4a 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -395,6 +395,7 @@ sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrJP
 
 # T11 / Juno Tablet 2
 sensor:modalias:acpi:NSA2513*:dmi:*:svnDigitek:*
+sensor:modalias:acpi:NSA2513*:dmi:*:svnDefaultstring:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; 1, 0, 0; 0, 0, 1
 
 #########################################
